front/components/assistant/AssistantDetails.tsx

```diff
@@ -251,6 +252,9 @@ export function AssistantDetails({
 
   const [showRestoreModal, setShowRestoreModal] = useState(false);
   const showEditorsTabs = assistantId != null && !isGlobalAgent;
+  const showAgentMemory = !!agentConfiguration?.actions.find(
+    (a) => a.name === "agent_memory"
```

Comment:
Can't people rename those ?

Reply:
Interesting side quest. Done

Reply:
Can technically be renamed (search allows that). Can'be be renamed for now for other tools but will be soon doable as we add ability to add more tools so great catch!

Reply:
(tested with internal MCP server and remote MCP server (stripe)) Didn't test clientSideMCPServer but code diff is low risk.